.circleci/prepare.sh

```diff
@@ -1,12 +1,28 @@
 #!/usr/bin/env bash
-sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get -y install libssl-dev python python-setuptools python3 python3-setuptools python3-pip git rsync psmisc
-cd $HOME && git clone https://github.com/cossacklabs/themis && cd themis && sudo make install
-cd $HOME && for version in $VERSIONS; do mkdir go_root_$version; cd go_root_$version; wget https://storage.googleapis.com/golang/go$version.linux-amd64.tar.gz; tar xf go$version.linux-amd64.tar.gz; cd -; done
 mkdir $HOME/$GOPATH_FOLDER
-cd $HOME && mkdir -p $HOME/$GOPATH_FOLDER/src/github.com/cossacklabs/themis; mkdir -p $HOME/$GOPATH_FOLDER/src/github.com/cossacklabs/acra; ln -s $HOME/themis/gothemis $HOME/$GOPATH_FOLDER/src/github.com/cossacklabs/themis/gothemis; cp -r $HOME/project/* $HOME/$GOPATH_FOLDER/src/github.com/cossacklabs/acra/
-cd $HOME && GOPATH=$HOME/$GOPATH_FOLDER go get -d github.com/cossacklabs/acra/...
+
+sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get -y install libssl-dev python python-setuptools python3 python3-setuptools python3-pip git rsync psmisc
+cd $HOME
+git clone https://github.com/cossacklabs/themis
+cd themis
+sudo make install
+cd $HOME
+rm -rf themis
+for version in $VERSIONS; do
+    mkdir go_root_$version;
+    cd go_root_$version;
+    wget https://storage.googleapis.com/golang/go$version.linux-amd64.tar.gz;
+    tar xf go$version.linux-amd64.tar.gz;
+    cd -;
```

Comment:
I think it would be better to use `cd $HOME` instead of `cd -` for consistency. (Or vice versa.)

Alternatively, we could use the following on lines 12–13:
```
mkdir $HOME/go_root_$version
cd $HOME/go_root_$version
```

Finally, we don't have to write trailing semicolons (`;`) now that we use newlines.